src/Pool.sol

```diff
@@ -587,140 +533,98 @@ abstract contract Pool {
-    /// If zero, funding is stopped.
-    /// @param senderAddr The address of the sender
-    /// @return amt The target amount to be sent every second
-    function getAmtPerSec(address senderAddr) public view returns (uint128 amt) {
-        return senders[senderAddr].amtPerSec;
-    }
-
-    /// @notice Gets the target amount sent every second for the provided sub-sender.
-    /// The actual amount sent every second may differ from the target value.
-    /// It's rounded down to the closest multiple of the sum of the weights of
-    /// the sub-sender's receivers and split between them proportionally to their weights.
-    /// Each receiver then receives their part from the sub-sender's balance.
-    /// If zero, funding is stopped.
-    /// @param senderAddr The address of the sender
-    /// @param subSenderId The id of the sender's sub-sender
-    /// @return amt The target amount to be sent every second
-    function getAmtPerSecSubSender(address senderAddr, uint256 subSenderId)
-        public
-        view
-        returns (uint128 amt)
-    {
-        return subSenders[senderAddr][subSenderId].amtPerSec;
-    }
-
     /// @notice Sets the fraction of received funds to be dripped by the sender.
     /// @param sender The updated sender
     /// @param dripsFraction The fraction of received funds to be dripped.
-    /// Must be a value from 0 to `DRIPS_FRACTION_MAX` inclusively,
-    /// where 0 means no dripping and `DRIPS_FRACTION_MAX` dripping everything.
+    /// Must be a value from 0 to `MAX_DRIPS_FRACTION` inclusively,
+    /// where 0 means no dripping and `MAX_DRIPS_FRACTION` dripping everything.
     function _setDripsFraction(Sender storage sender, uint32 dripsFraction) internal {
-        require(dripsFraction <= DRIPS_FRACTION_MAX, "Drip fraction too high");
+        require(dripsFraction <= MAX_DRIPS_FRACTION, "Drip fraction too high");
         sender.dripsFraction = dripsFraction;
     }
 
     /// @notice Gets the fraction of received funds to be dripped by the provided user.
     /// @param userAddr The address of the user
     /// @return dripsFraction The fraction of received funds to be dripped.
-    /// A value from 0 to `DRIPS_FRACTION_MAX` inclusively,
-    /// where 0 means no dripping and `DRIPS_FRACTION_MAX` dripping everything.
+    /// A value from 0 to `MAX_DRIPS_FRACTION` inclusively,
+    /// where 0 means no dripping and `MAX_DRIPS_FRACTION` dripping everything.
     function getDripsFraction(address userAddr) public view returns (uint32 dripsFraction) {
         return senders[userAddr].dripsFraction;
     }
 
-    function _assertCurrReceivers(Sender storage sender, ReceiverWeight[] calldata currReceivers)
+    /// @notice Asserts that the list of receivers is the sender's currently used one.
+    /// @param sender The sender
+    /// @param currReceivers The list of the user's current receivers.
+    function _assertCurrReceivers(Sender storage sender, Receiver[] calldata currReceivers)
```

Comment:
Why not public?

Reply:
Because it accepts `Sender storage sender`

Reply:
Ah, i see